packages/api/src/models/transaction-wrapper.ts

```diff
@@ -16,7 +16,7 @@ import { BaseModel } from "./_default";
  */
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export async function executeOnDBTx<T extends Model<any, any>, X>(
-  model: BaseModel<T>,
+  model: BaseModelNoId<T>,
```

Comment:
There are some models that use this function that are based on the BaseModel with an id no?

Reply:
`BaseModel` extends `BaseModelNoId`, so as long as the function works w/o accessing the `id`, it can work on a more abstract representation of an entity - and it allows for models that rely on the no-id version to use the same function.